### PR TITLE
Update to Extreme Tux Racer 0.8.0 and update runtime/shared-modules

### PR DESCRIPTION
**Pull request by Calinou**
This also upgrades SFML to the latest version and switches to CMake with Ninja for faster builds.

Tested on my local Fedora 31 setup, works perfectly :slightly_smiling_face:

### CODE REVIEW

**Comment by bilelmoussaoui** [comment]
If possible, can you run `appstream-util upgrade` on the appdata file? it's still using the old format :) thank you!

File: etr.appdata.xml [hunk 1]
```diff
@@ -1,32 +1,45 @@
...
-        <screenshot width="800" height="600">https://a.fsdn.com/con/app/proj/extremetuxracer/screenshots/ETR3.png</screenshot>
-        <screenshot width="800" height="600">https://a.fsdn.com/con/app/proj/extremetuxracer/screenshots/ETR2.png</screenshot>
-    </screenshots>
-    <url type="homepage">http://extremetuxracer.sourceforge.net/</url>
-    <!-- upstream doesn't yet have this, it's maintained in Fedora -->
-    <updatecontact>nils@redhat.com</updatecontact>
-</application>
+<?xml version="1.0" encoding="UTF-8" ?>
+<component type="desktop">
+  <id>etr.desktop</id>
```
Comment (bilelmoussaoui) [inline etr.appdata.xml]:
```suggestion
  <id>net.sourceforge.ExtremeTuxRacer</id>
  <launchable type="desktop-id">net.sourceforge.ExtremeTuxRacer.desktop</launchable>
```

**Comment by bilelmoussaoui** [comment]
Thank you for the fast updates :)

**Comment by Calinou** [comment]
@bilelmoussaoui Thanks for reviewing this PR quickly as well :slightly_smiling_face:

**Pull request merged**